src/service.rs

```diff
@@ -151,6 +173,29 @@ pub struct ServiceInfo {
     pub account_password: Option<OsString>,
 }
 
+impl ServiceInfo {
+    pub fn from_raw(raw: winsvc::QUERY_SERVICE_CONFIGW) -> Result<ServiceInfo> {
+        Ok(ServiceInfo {
+            name: OsString::from(""),
```

Comment:
i didn't want to pollute the code with new types, but yes there are less fields returned that used in `ServiceInfo` so a new type is probably warranted.

Reply:
That's totally fine to create a separate type which suits the result the most. Don't worry, we won't reject the PR because you've added a new type 👍 Thanks for contributing.

Reply:
i extracted this out into `ServiceConfig`